packages/docprovider/src/ydrive.ts

```diff
@@ -21,10 +21,19 @@ import {
   ISharedModelFactory
 } from '@jupyter/collaborative-drive';
 import { Awareness } from 'y-protocols/awareness';
+import { ISettingRegistry } from '@jupyterlab/settingregistry';
+import * as encoding from 'lib0/encoding';
 
 const DISABLE_RTC =
   PageConfig.getOption('disableRTC') === 'true' ? true : false;
 
+const SAVE_MESSAGE = (() => {
+  const encoder = encoding.createEncoder();
+  encoding.writeVarUint(encoder, 2);
```

Comment:
Since we have a named constant on the Python side, maybe we could have something similar on the JS side instead of `2`?